src/apps/Settings/Settings.js

```diff
@@ -184,31 +175,7 @@ class Settings extends React.Component {
               </Field>
             </Option>
           )}
-          <Option
-            name="Home Page"
-            text={`The default home page is a list of shortcuts to other apps. You can set another home page, which can be any of your existing apps. You might want to install the Home app first and select that!`}
-          >
-            <WideFlex>
-              <Field label="Select app">
-                <DropDown
-                  active={reducedAppNames.indexOf(selectedHomeApp)}
-                  items={reducedAppNames}
-                  onChange={this.handleHomeAppChange}
-                  wide
-                />
-              </Field>
-              <Field label="Enter tab name">
-                <TextInput
-                  onChange={this.handleHomeNameChange}
-                  value={homeAppName}
-                  wide
-                />
-              </Field>
-            </WideFlex>
-            <Button mode="strong" onClick={this.handleHomeSettingsSave}>
-              Submit Changes
-            </Button>
-          </Option>
+          <HomeSettings {...this.props} />
```

Comment:
good, but, does HomeSettings need all props from this component?

Reply:
No, you're right. It will be better just with needed ones.